api/v1/mongodbcommunity_types.go

```diff
@@ -489,6 +490,13 @@ func (m MongoDBCommunity) GetMongoDBVersion() string {
 	return m.Spec.Version
 }
 
+// GetMongoDBVersionForAnnotation returns the MDB version used to annotate the object.
+// Here it's the same of GetMongoDBVersion, but a different name is used in order to make
```

Comment:
Did you mean "same as GetMongoDBVersion"?

Reply:
Whoops some Italian construct crept into my English :D thanks for noticing!